### ISSUE TITLE
[bug]error literal parse after comments

### ISSUE THREAD

**Issue by songhlc** [acrazing/html5parser#7]
version: 1.1.2

source code
```
<!-- it is comment -->
-
```
expect
```
node: {
   body: [{
        type: 'Literal',
        value: 'it is comment'
   }]
}
```

actual result:
```
node: {
   body: [{
        type: 'Literal',
        value: 'it is comment \n-\n'
   }]
}
```

**Comment by songhlc** [comment]
```
function parseOpenTag() {
...
while (++index < count) {
          token = tokens[index];
          if (token.type === TokenKind.OpenTagEnd) {
            **node = void 0;  //  set node to undefined to fix bug**
            break;
          }
          appendLiteral();
        }
...
}
```

**Comment by acrazing** [comment]
Fixed in `html5parser@1.2.0`, thank you.
